apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.html

```diff
@@ -87,15 +87,19 @@
                                 {{ commonLocaleData?.app_from }}
                             </th>
                             <td mat-cell *matCellDef="let element">
-                                {{ element?.taxPeriod?.from ?? "-" }}
+                                {{
+                                    element?.taxPeriod?.from
+                                        ? dayjs(element.taxPeriod.from).format(giddhDateFormat)
```

Comment:
revert code for date format in overall file and assign to api developer as confirmed with kriti

Reply:
Done